easybuild/easyconfigs/w/waLBerla/waLBerla-6.1_fix_cmakelist_for_easybuild.patch

```diff
@@ -0,0 +1,12 @@
+diff -ruN walberla-v6.1.orig/CMakeLists.txt walberla-v6.1/CMakeLists.txt
+--- walberla-v6.1.orig/CMakeLists.txt	2023-10-17 13:06:02.619565000 +0200
++++ walberla-v6.1/CMakeLists.txt	2023-10-18 14:47:35.047985995 +0200
+@@ -1314,7 +1314,7 @@
+                  src/waLBerlaDefinitions.h    )
+ 
+ install( FILES ${walberla_BINARY_DIR}/src/waLBerlaDefinitions.h DESTINATION walberla/ )
+-
++install( DIRECTORY ${walberla_BINARY_DIR}/apps/ DESTINATION walberla/ )
```

Comment:
I think we should review what gets installed. If we don't build the benchmarks, tutorials, tools, showcases etc, we also don't need to install them. We could just just start with installing the headers & python interface (i.e. `pythonmodule` directory). If that's not enough, we'll hear about it later from people who actually want to use it :)

Reply:
And we can install in `.` instead of `walberla`, it strips one level of nesting which is not really needed.